01-node-tutorial/answers/writeWithPromisesThen.js

```diff
@@ -0,0 +1,70 @@
+const { writeFile, readFile } = require("fs").promises;
+
+// async function writer() {
```

Comment:
It's not typically good practice to leave commented code in your pull requests. If it is code you'd like to keep, I would recommend creating a new branch for this if you ever want to refer to it again.